src/qdisp/CzarStats.h

```diff
@@ -123,15 +127,14 @@ class TimeCountTracker {
 public:
```

Comment:
This utility class doesn't really belong to this header (`qdisp/CzarStats.h`). I'd rather put it into the same header where you define `TIMEPOINT` and `CLOCK`. And my suggestion is to put `TIMEPOINT`, `CLOCK`, and `TimeCountTracker` into the dedicated header `global/clock_defs.h`.

In general, we should be aiming at having each namespace-scoped class in the dedicated header. Exceptions are - utility headers shared by many other dependent classes, nested classes defined within other classes, or implementation-specific classes in the *.CC files.